packages/@ama-sdk/schematics/schematics/typescript/shell/templates/base/.yarnrc.yml

```diff
@@ -0,0 +1,6 @@
+nodeLinker: pnp
+
+packageExtensions:
+  "@ama-sdk/generator-sdk@*":
```

Comment:
do we keep this reference when we actually use the new generator?

Reply:
change to @ama-sdk/schematics